lib/spotlight/engine.rb

```diff
@@ -189,6 +189,7 @@ def self.blacklight_config
     config.ga_email = nil
     config.ga_analytics_options = {}
     config.ga_page_analytics_options = config.ga_analytics_options.merge(limit: 5)
+    config.ga_anonymize_ip = false # false for backwards compatibility
```

Comment:
should this also be added to the template configuration?

Reply:
Probaby.. just rebased and pushed up this change. Nice catch!